.circleci/config.yml

```diff
@@ -116,6 +116,28 @@ workflows:
             branches:
               only: master
 
+      - build:
```

Comment:
Please make this change in config.yml.j2 then run the script in that directory.

Reply:
i did this way, i've added only 1 new dir, so no need to change config.yaml.j2
this loop take care on it dynamically: https://github.com/astronomer/ap-vendor/blob/b5bc6155f13d79d810a4c4ccc19a7d11c76d21b2/.circleci/config.yml.j2#L6-L5

Reply:
ah right! thank you